fr/integrations/plugin-packs/procedures/cloud-cadvisor-api.md

```diff
@@ -0,0 +1,190 @@
+| <container_name_or_id>#container.swap.usage.bytes                  | SWAP memory usage              |   B    |
+
+By default, the --use-name flag will instance the metric with the <container_name>. 
+If you want to use the <container_id> instead, remove it from the EXTRAOPTIONS macro at the Service Template level.
+
+<!--Container-Disk-IO-->
+
+| Metric name                                                   | Description                      | Unit   |
+| :-------------------------------------------------------------| :------------------------------- | :----- |
+|	<container_name_or_id:device>#disk.io.read.bytespersecond     | Disk I/O Read from the container |  B/s   |
+| <container_name_or_id:device>#disk.io.write.bytespersecond    | Disk I/O Read from the container |  B/s   |
+
+By default, the --use-name flag will instance the metric with the <container_name>. 
+If you want to use the <container_id> instead, remove it from the EXTRAOPTIONS macro at the Service Template level.
+
+<!--Container-Traffic-->
+
+| Metric name                                                            | Description                      | Unit   |
+| :----------------------------------------------------------------------| :------------------------------- | :----- |
+|	<container_name_or_id.network_int>#container.traffic.in.bitspersecond  | Container incoming traffic       |  bps   |
+| <container_name_or_id.network_int>#container.traffic.out.bitspersecond | Container outgoing traffic       |  bps   |
+
+By default, the --use-name flag will instance the metric with the <container_name>. 
+If you want to use the <container_id> instead, remove it from the EXTRAOPTIONS macro at the Service Template level.
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+### cAdvisor
+
+Un conteneur ou un démon cAdvisor doit être en cours d'execution et disponible. La 
+documentation officielle permet de [déployer le nécessaire rapidement](https://github.com/google/cadvisor#quick-start-running-cadvisor-in-a-docker-container).
+
+### Flux réseaux
+
+Le Collecteur doit être en mesure de contacter le serveur hébergeant cAdvisor au travers 
+du port TCP/8080 port. Attention, selon la configuration le port peut être différent. 
+
+## Installation
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online IMP Licence & IT-100 Editions-->
+
+1. Installer le Plugin sur tous les Collecteurs Centreon allant superviser des containers via cAdvisor:
+
+```bash
+yum install centreon-plugin-Cloud-cAdvisor-Api
+```
+
+2. Dans l'interface de Centreon, installer le Plugin Pack *cAdvisor API* depuis la page "Configuration > Plugin Packs > Manager"
+
+<!--Offline IMP License-->
+
+1. Installer le Plugin sur tous les collecteurs Centreon devant superviser des containers via cAdvisor:
+
+```bash
+yum install centreon-plugin-Cloud-cAdvisor-Api
+```
+
+2. Installer le Plugin sur chaque Collecteur allant superviser des containers via cAdvisor:
```

Comment:
Points 1 and 2 are the same, apart from some lowercase/uppercase characters. Is that intended?